src/components/form/inputs/TextAreaInputNew.tsx

```diff
@@ -94,8 +94,8 @@ export default function TextAreaInputNew({
           setShowPlaceholder(false);
         }}
         onBlur={() => {
-          setCharacterError(false);
-          setColor && setColor(false, name);
+          // setCharacterError(false);
+          // setColor && setColor(false, name);
```

Comment:
please delete instead of commenting, do any other components need this functionality since this is a common component